docs/UserGuide.md

```diff
@@ -176,31 +273,105 @@ and at the bottom of the window, is a link to our user guide.
 * Projects, as well as their members, tasks to complete in the project, and other important details for managing your project are stored here!
 
 ## Quick start
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```

Comment:
Perhaps the warning or danger symbol/label from the navigation guide could be used here instead